HISTORY.rst

```diff
@@ -38,6 +38,12 @@ Unreleased Changes
 ------------------
 * General
     * Update to FontAwesome 5 icons in the QT interface.
+    * In response to the deprecation of ``setup.py``-based commands in Python
+      3.10, the recommended way to build python distributions of
+      ``natcap.invest`` is now with the ``build`` package, and installation
+      should be done via ``pip``.  The ``README`` has been updated to reflect
+      this change, and this should only be noticeable for those installing
+      ``natcap.invest`` from source.
```

Comment:
Could we also note the bugfix in `make install`? e.g. `make install` now always installs the current version of ``natcap.invest``, whereas previously it installed from whichever ``whl`` was found with latest version.

Reply:
Yep!  Patched in c71f02d7